x11-libs/qscintilla/qscintilla-2.13.3.ebuild

```diff
@@ -56,24 +73,41 @@ qsci_run_in() {
 }
 
 src_configure() {
-	if use designer; then
-		# prevent building against system version (bug 466120)
-		append-cxxflags -I../src
-		append-ldflags -L../src
-	fi
+	my_src_configure() {
+		if use designer; then
+			# prevent building against system version (bug 466120)
+			append-cxxflags -I../src
+			append-ldflags -L../src
+		fi
```

Comment:
this can/should probably be moved outside `my_src_configure`

Reply:
:+1: